### PR TITLE
Change TensorRT-LLM

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

**Comment by krishung5** [comment]
@mc-nv I think the changes in this PR are merged within other previous PR. Should we close this one?

File: build.py [hunk 1]
```diff
@@ -1108,14 +1094,7 @@ def create_dockerfile_linux(
 RUN python3 -m pip install --upgrade pip \\
       && pip3 install transformers
 
-# Drop the static libs
-RUN ARCH="$(uname -i)" \\
-      && rm -f ${TRT_ROOT}/targets/${ARCH}-linux-gnu/lib/libnvinfer*.a \\
-          ${TRT_ROOT}/targets/${ARCH}-linux-gnu/lib/libnvonnxparser_*.a
-
```
Comment (krishung5) [inline build.py:1115]: Why removing these lines? Is dropping libs causing some issues?

Comment (mc-nv) [reply build.py:1115]: Removing of those libs doesn't make our container image smaller if that's what is it designed for.
Is the removing of those libs has any other logical purpose?

Comment (krishung5) [reply build.py:1115]: Thought it would reduce the container size.. Thanks for the explanation.

**Review by krishung5 — APPROVED** [review]
LGTM! Might need to rebase onto main branch.

**Pull request merged**